### PR TITLE
Enable ETW manifests for ChakraCore

### PR DESCRIPTION
**Pull request by digitalinfinity**

### CODE REVIEW

**Comment by digitalinfinity** [comment]
@dotnet-bot test Ubuntu ubuntu_linux_release_static please

**Comment by digitalinfinity** [comment]
@tcare please review.

File: jenkins/check_copyright.sh [hunk 1]
```diff
@@ -34,6 +34,7 @@ git diff --name-only `git merge-base origin/master HEAD` HEAD |
     grep -v -E '\.def$' |
     grep -v -E '\.inc$' |
     grep -v -E '\.json$' |
+    grep -v -E '\.man$' |
```
Comment (tcare) [inline jenkins/check_copyright.sh]: Good catch

**Comment by tcare** [comment]
Looks good!

**Comment by digitalinfinity** [comment]
Thanks for the review!

**Comment by digitalinfinity** [comment]
Fixes Microsoft/ChakraCore#328

**Pull request merged**